### PR TITLE
Added @profile on several key methods in the Android lifecycle, refactored by extracting into methods a little

### PR DESCRIPTION
**Pull request by PanayotCankov**

### CODE REVIEW

**Comment by dtopuzov** [comment]
run ci

File: tns-core-modules/ui/frame/frame.android.ts [hunk 1]
```diff
@@ -641,22 +639,52 @@ class ActivityCallbacksImplementation implements AndroidActivityCallbacks {
...
 
-            rootView = frame;
+    @profile
+    private setupUI(activity: android.app.Activity): void {
+        // Initialize native visual tree;
+        this._rootView._setupUI(activity);
+    }
+
+    @profile
+    private setContentView(activity: android.app.Activity): void {
```
Comment (hshristov) [inline tns-core-modules/ui/frame/frame.android.ts]: Why do we pass `activity` here?

**Comment by PanayotCankov** [comment]
run ci

**Comment by dtopuzov** [comment]
run ci

**Comment by PanayotCankov** [comment]
run ci

**Pull request merged**